src/pages/certification/components/AuditorRunTestForm/AuditorRunTestForm.tsx

```diff
@@ -51,9 +51,19 @@ const AuditorRunTestForm: React.FC<IAuditorRunTestForm> = ({
 }) => {
 
   const dispatch = useAppDispatch();
+  const confirm = useConfirm();
+
+  const { repoUrl } = useAppSelector((state) => state.certification);
+  const { profile } = useAppSelector((state) => state.profile);
+  const { showConfirmConnection, accessStatus, accessToken, verifying } = useAppSelector((state) => state.repoAccess);
+  
   const form: any = useForm({
     schema: auditorRunTestFormSchema,
     mode: "all",
+    defaultValues: profile && profile.dapp ? {
+      repoURL: profile.dapp.owner && profile.dapp.repo ? `https://github.com/${profile.dapp.owner}/${profile.dapp.repo}` : undefined,
+      name: profile.dapp.name, version: profile.dapp.version,
+    } : undefined
```

Comment:
@EehMauro , FYI - need not always set defaultValues for Testing form now as auditors might want to test different dapps. 'Test again' offers them to retest the previously tested dapp, 'New Test' offers them to test a new dapp. Later, when multiple dapps are to be saved in the auditor profile, and they pick a default dapp , maybe it would be pre-populated for testing. 

I will correct this in PLT-7787 respective PR

@RSoulatIOHK this is my understanding. Please correct if otherwise.

Reply:
ignore this comment, 

we will prefil the testing form with values within dapp profile